docs/package.json

```diff
@@ -11,11 +11,8 @@
     "lintfix": "node .. run lint -- --fix",
     "snap": "tap",
     "test": "tap",
-    "pretest": "node .. run rebuild-cmark",
```

Comment:
These were added as a stop-gap to prevent docs that would eventually throw an exception when the docs site itself went to build these files.  Is what the cli does now identical to the docs site so that we are getting the same guard?

Reply:
Yes, the behavior is the same. The only different is we no longer need to rebuild any native dependencies since `remark` is pure JS.